### PR TITLE
update factories to interop-config 1.0

### PR DESCRIPTION
**Pull request by sandrokeil**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "container-interop/container-interop": "^1.1",
         "fabpot/php-cs-fixer": "^1.10",
         "phpunit/phpunit": "^4.8 || ^5.2",
-        "sandrokeil/interop-config": "^0.3.1",
+        "sandrokeil/interop-config": "^1.0",
```
Comment (prolic) [inline composer.json:43]: conflict-section missing

**Comment by prolic** [comment]
Can you fix those php-cs issues, too?

**Comment by sandrokeil** [comment]
@prolic Good catch. :+1:

**Comment by sandrokeil** [comment]
@prolic The php-cs settings are different from other repos. Have we changed the rules?

**Comment by prolic** [comment]
No. Just some inconsistencies

**Comment by sandrokeil** [comment]
Fixed but it's interesting that some tests failed in a Docker environment. There is a timestamp issue, but I think it's out of scope of this package.

**Comment by prolic** [comment]
Please create an issue, I'll will it, when I have some time.

Sascha-Oliver Prolic

2016-03-16 20:34 GMT+00:00 Sandro Keil notifications@github.com:

**Pull request merged**